tests/test_scripts.py

```
@@ -55,12 +65,59 @@ def _run_script(script_name: str, csv_dir: str, output_dir: str) -> subprocess.C
+                          "impulse_response"]
+        for stem in expected_stems:
+            for ext in ("png", "pdf"):
+                assert os.path.exists(os.path.join(outdir, f"{stem}.{ext}")), (
+                    f"Missing {stem}.{ext}")
+
+
+def test_plot_precision_new_cli_flags(csv_dir):
+    """Issue #12-spec'd invocation: --input-dir / --output-dir."""
+    script_path = Path(__file__).parent.parent / "scripts" / "plot_precision.py"
+    with tempfile.TemporaryDirectory() as outdir:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)